recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/mbop/MBOPService.java

```diff
@@ -43,6 +43,7 @@ MBOPUsers getUsersByOrgId(
         @RestQuery("limit")                             int limit,
         @RestQuery("offset")                            int offset,
         @RestQuery("include_permissions")               boolean includePermissions,
-        @RestQuery("status")                            String status
+        @RestQuery("status")                            String status,
+        @RestQuery("skipTotalCount")                    boolean skipTotalCount
```

Comment:
We're adding a new `skipTotalCount` field but it doesn't seem to be used anywhere. Is that normal?